### ISSUE TITLE
laboratoriotutkimus

### ISSUE THREAD

**Issue by Finto-ehdotus** [Finto-ehdotus/YSE#13147]
**Käsitteen tyyppi**

Muutos olemassa olevaan käsitteeseen

**prefabel**

[laboratoriotutkimus](http://www.yso.fi/onto/yso/p6757)

**Tila**

Käsittelyssä

**Ehdotettu muutos**

Termin merkityksen tarkennus tms? Viittaako tieteelliseen tutkimukseen joka tehdään laboratorioympäristössä (sv. forskning), vai lääketieteelliseen "toimenpiteeseen" (sv. undersökning)?

**Perustelut ehdotukselle**

Yläkäsite on nyt tutkimus=forskning=research, siirretäänkö?

**Ehdottajan organisaatio**

ALLFO

**Comment by mirmaid** [comment]
Ei ainakaan ole Melindassa rajautunut lääketieteeseen (eikä kai tarkoituskaan). Suomeksi ei ole niin helppo tätä eroa viitteistä hahmottaa, kun suomeksi tutkimus tietysti sopii hyvin molempiin sekä forskning että undersökning.
esim. 
https://melinda.kansalliskirjasto.fi/byid/019018817
https://melinda.kansalliskirjasto.fi/byid/018735490
https://melinda.kansalliskirjasto.fi/byid/018847893
https://melinda.kansalliskirjasto.fi/byid/018995846
https://melinda.kansalliskirjasto.fi/byid/018999931

Onko näille ruotsiksi sama vai eri vastineita?

**Comment by mariaforsen** [comment]
Ruotsiksi siis kaksi eri termiä eri merkityksellä, forskning (tieteellinen tutkimus) vs. undersökning (esim. laboratoriokokeet https://finto.fi/yso/fi/page/p6756 ja muut konkreettiset tutkimukset, riippumatta siitä ovatko lääketieteellisellä tai muilla aloilla tapahtuvia)

https://melinda.kansalliskirjasto.fi/byid/019018817 -näyttää olevan undersökning (jos mikään näistä?) kun aiheena on asbestiin liityvien laboratoriotutkimusten vertailu 
https://melinda.kansalliskirjasto.fi/byid/018735490 -näyttää olevan undersökning, eli aiheena laboratorioissa tehtäviä tutkimuksia esim. verensiirtoihin liittyvät laboratoriotutkimukset
https://melinda.kansalliskirjasto.fi/byid/018847893 -näyttää olevan undersökning, kun taas vertaillaan eri laboratorioiden testituloksia
https://melinda.kansalliskirjasto.fi/byid/018995846 -laboratoriotutkimuksia ei mainita, olisiko kyseessä siis tieteellinen tutkimus joka tapahtuu laboratorioissa eli forskning
https://melinda.kansalliskirjasto.fi/byid/018999931 -näyttää olevan undersökning taas kun vertaillaan laboratoriotutkimusten tuloksia

Suurin ongelma on kuitenkin hierarkia, eli kun tutkimus viittaa researchiin eli forskning, niin tämä ei sovi alakäsitteeksi, jos voi viitata mihin tahansa laboratoriotutkimukseen?

**Comment by mirmaid** [comment]
^ No, on tämä ehkä myös isompi asia kuin vain hierarkia. Sanastossa on myös _laboratoriokokeet_, ja sitäkin on käytetty monenlaisissa yhteyksissä. Suomeksi yksikkö "tutkimus" tuntuisi viittaavan tutkimustoimintaan ja laboratoriokokeet yksittäisiin testeihin, mutta myös laboratoriotutkimuksen yhteydessä voidaan tehdä erillisiä kokeita. En oikeastaan tiedä onko "laboratoriotutkimus" = "laboratorio-oloissa tehtävä tutkimus" ihan tätä päivää terminä.

https://fi.wikipedia.org/wiki/Laboratoriotutkimus

**Comment by mariaforsen** [comment]
Joo, kun katsoo yllä mainittua wikipedia-artikkelia, niin näyttää siltä että siinä ainakin laboratoriotutkimus=laboratoriokoe, eli päällekkäisyyttä http://www.yso.fi/onto/yso/p6756 ? Ruotsin vastinesivu onkin laboratorieprov.

**Comment by mirmaid** [comment]
Pitäisikö tehdä esim. ohjaus laboratoriotutkimukset (laboratoriokokeet) KÄ laboratoriokokeet? Tosin suomeksi taitaa laboratoriotutkimus olla nykyään paljon käytetympi näistä undersökning-tyylisistä kokeista...mutta jos noita korjailisi ensin kokeet-suuntaan soveltuvin osin ja sitten katsoisi onko mitään forskning-tutkimusta tarpeen säilyttää, ja jos ei, vaihtuisiko päätermi suomeksi > tutkimukset.

**Comment by mirmaid** [comment]
^ Tehdään ehdotettu korjaus, jatketaan käsittelyä sen jälkeen.

**Comment by mirmaid** [comment]
Uusi ehdotus: yhdistetään laboratoriokokeet ja laboratoriotutkimus > laboratoriotutkimukset.

**Comment by YSO-kokous** [comment]
**Päätös**

Toimitaan yllä olevan ehdotuksen mukaisesti ja yhdistetään käsitteet.

**Comment by mariaforsen** [comment]
Sv. laboratorieundersökningar HT laboratorieprov, laboratorietest
